index.js

```diff
@@ -1,28 +1,100 @@
-const toggleAnswerButton = document.querySelector(".js-btn-answer");
-const showAnswer = document.querySelector(".js-btn-showAnswer");
-const hideAnswer = document.querySelector(".js-btn-hideAnswer");
-const showAnswerH = document.querySelector(".js-answer-heading ");
-const hideAnswerT = document.querySelector(".js-answer-text");
-const bookmark = document.querySelector(".js-bookmark");
-const homebtn = document.querySelector(".js-home-btn");
-const bookmarkbtn = document.querySelector(".js-bookmark-btn");
-
-toggleAnswerButton.addEventListener("click", () => {
-  showAnswer.classList.toggle("hidden");
-  hideAnswer.classList.toggle("hidden");
-  showAnswerH.classList.toggle("hidden");
-  hideAnswerT.classList.toggle("hidden");
+const cardSet = document.querySelectorAll("[data-js=card-set]");
+
+cardSet.forEach((card) => {
+  const bookmark = card.querySelector("[data-js=bookmark]");
+  const AnswerBtn = card.querySelector("[data-js=btn-answer]");
+  const showAnswer = card.querySelector("[data-js=showAnswer]");
+  const hideAnswer = card.querySelector("[data-js=hideAnswer]");
+  const answerH = card.querySelector("[data-js=Answer-heading]");
```

Comment:
Is the saving of a few letters really worth the unintuitiveness it adds to your code? Dont get me wrong its up to you, but i would get confused then i would name many variables like that.